p8e-sdk/src/main/kotlin/io/p8e/ContractManager.kt

```diff
@@ -328,17 +328,45 @@ class ContractManager(
 //            }
 //    }
 
+
+    /**
+     * Returns an existing Contract object that can be configured and executed.
+     *
+     * @param contractClazz subclass of P8eContract that represents the original contract
+     * @param executionUuid the execution id of the original contract
+     * @param isFragment flag identifying if the original contract execution was requested (true) or initiated (false)
```

Comment:
This isFragment param comment confuses me, but I'm not entirely sure how to make it better 😆

Reply:
maybe something like 'flag identifying whether this contract is to be a response to a previous execution request (true) or a fresh execution (false)'